examples/dotnet/Examples/Relationships.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using MongoDB.Bson;
+using Realms;
+
+namespace Examples
+{
+    public class OneToOneRelationship
+    {
+        // :code-block-start: one-to-one
+        public class Person : RealmObject
+        {
+            public string Name { get; set; }
+            public DateTime Birthdate { get; set; }
```

Comment:
```suggestion
            public DateTimeOffset Birthdate { get; set; }
```

Does this not result in errors during compilation? `DateTime` is not a valid Realm type and I thought we'd error out if we encounter it.